idl/storage_service.idl.hh

```diff
@@ -24,10 +24,15 @@ struct table_load_stats final {
     int64_t split_ready_seq_number;
 };
 
-struct load_stats final {
+struct load_stats_v1 final {
     std::unordered_map<::table_id, locator::table_load_stats> tables;
 };
 
+struct load_stats {
+    std::unordered_map<::table_id, locator::table_load_stats> tables;
+    std::unordered_map<locator::host_id, uint64_t> capacity;
```

Comment:
Can't we just append an optional member for backward compatibility?
Why do we need a new verb?

Reply:
It's mentioned in the commit description: the object is marked final in IDL.

Reply:
Why a map? Isn't the RPC issued to a single node?

Reply:
I guess it's because of load_stats operator+=. But there's no need for the rpc payload to be the same as the accumulator. We could have global_load_stats(host_id, load_stats) to transform.